src/match_stream.erl

```diff
@@ -3,7 +3,7 @@
 %% @copyright (C) 2011 InakaLabs SRL
 %% @doc Match Stream main interface
 %% @end
-%%-------------------------------------------------------------------
+%%----------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Line 6 is too long: %%----------------------------------------------------------------------------------------------------------------------------------.